UnitTestFrameworkPkg/GoogleTest.md

```diff
@@ -872,6 +920,57 @@ extern "C" {
 }
 ```
 
+#### FunctionMockLib Mocks - Definition for non-Libraries
+The definitions of mock functions for non-libraries, or interfaces that don't have an
+inf associated with them (for example, protocols or PPIs) can be in the same header file
+as the declarations.
+
+Below the struct of mock function declarations, you can use `MOCK_INTERFACE_DEFINITION`
+and `MOCK_FUNCTION_DEFINITION`.
+For example, when creating the definition of the mock for `RngProtocol`,
+the definitions below are made in `MockRng.h`
+
+```cpp
+...
+
+MOCK_INTERFACE_DEFINITION(MockEfiRngProtocol);
+
+MOCK_FUNCTION_DEFINITION(MockEfiRngProtocol, GetInfo, 3, EFIAPI);
+MOCK_FUNCTION_DEFINITION(MockEfiRngProtocol, GetRng, 4, EFIAPI);
+```
+
+To allow multiple instances of the protocol to be made, we create a macro in the same header file following the naming convention: `MOCK_<INTERFACE_NAME>_INSTANCE`.

```

Comment:
To allow multiple instances of the mocked protocol to be created, a macro in the same header file ...